.github/workflows/build-yocto.yml

```diff
@@ -141,4 +141,10 @@ jobs:
         id: print-output
         run: |
           echo "Downloads URL: https://quic-yocto-fileserver-1029608027416.us-central1.run.app/${GITHUB_RUN_ID}/"
```

Comment:
maybe also you can change the echo to use the ${BASE_ARTIFACT_URL}

Reply:
Good catch, let me fix that.